WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -263,7 +264,40 @@ public static void SortByAmount(this TxOutList list)
 		{
 			list.Sort((x, y) => x.Value.CompareTo(y.Value));
 		}
+    
+		/// <param name="startWithM">The keypath will start with m/ or not.</param>
+		/// <param name="format">h or ', eg.: m/84h/0h/0 or m/84'/0'/0</param>
+		public static string ToString(this KeyPath me, bool startWithM, string format)
+		{
+			var toStringBuilder = new StringBuilder(me.ToString());
+
+			if (startWithM)
+			{
+				toStringBuilder.Insert(0, "m/");
+			}
+
+			if (format == "h")
```

Comment:
It seems `format` is `h` or not. If that is correct then consider using a boolean.

Reply:
I am not sure about this. Maybe Adam will add more formats later.